fix: skip app template loading when DISABLED_EXTENSIONS is set (#7145)
shopware/platform: changelog/_unreleased/2025-02-27-exclude-app-templates-namespaces-on-disable_extensions.md

---

> title: Exclude app namespaces in BundleHierarchyBuilder when DISABLE_EXTENSIONS is set

Reviewer: ```suggestion
title: Exclude app namespaces in BundleHierarchyBuilder if DISABLE_EXTENSIONS is set
```

issue: 6842
---
# Core

> * Changed the `\Shopware\Core\Framework\Adapter\Twig\NamespaceHierarchy\BundleHierarchyBuilder` to exclude apps namespaces when `DISABLE_EXTENSIONS` environment variable is set.

Reviewer: ```suggestion
* Changed the `\Shopware\Core\Framework\Adapter\Twig\NamespaceHierarchy\BundleHierarchyBuilder` to exclude app namespaces if `DISABLE_EXTENSIONS` environment variable is set.
```
